examples/messengers/telegram/1_basic.py

```diff
@@ -0,0 +1,79 @@
+# %% [markdown]
+"""
+# 1. Basic
+
+The following example shows how to run a regular DFF script in Telegram.
+It asks users for the '/start' command and then loops in one place.
+"""
+
+
+# %%
+import os
+
+from dff.script import conditions as cnd
+from dff.script import labels as lbl
+from dff.script import RESPONSE, TRANSITIONS, Message
+from dff.messengers.telegram import PollingTelegramInterface, TelegramMessenger
+from dff.pipeline import Pipeline
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %% [markdown]
+"""
+In order to integrate your script with Telegram, you need an instance of
+`TelegramMessenger` class and one of the following interfaces:
+`PollingMessengerInterface` or `WebhookMessengerInterface`.
+
+`TelegramMessenger` encapsulates the bot logic.
+Like Telebot, `TelegramMessenger` only requires a token to run.
+However, all parameters from the Telebot class can be passed as keyword arguments.
+
+The two interfaces connect the bot to Telegram. They can be passed directly
+to the DFF `Pipeline` instance.
+"""
+
+
+# %%
+script = {
+    "greeting_flow": {
+        "start_node": {
+            RESPONSE: Message(),
+            TRANSITIONS: {"greeting_node": cnd.exact_match(Message(text="/start"))},
+        },
+        "greeting_node": {
+            RESPONSE: Message(text="Hi"),
+            TRANSITIONS: {lbl.repeat(): cnd.true()},
+        },
+        "fallback_node": {
+            RESPONSE: Message(text="Please, repeat the request"),
+            TRANSITIONS: {"greeting_node": cnd.exact_match(Message(text="/start"))},
+        },
+    }
+}
+
+# testing
+happy_path = (
+    (Message(text="/start"), Message(text="Hi")),
+    (Message(text="Hi"), Message(text="Hi")),
+    (Message(text="Bye"), Message(text="Hi")),
+)
+
+
+# %%
+messenger = TelegramMessenger(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+interface = PollingTelegramInterface(messenger=messenger)
```

Comment:
Can we make `PollingTelegramInterface` signature easy ?
For exampel `PollingTelegramInterface(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))`.
`TelegramMessenger` seems redundant for ordinal usage

Reply:
It's should be done for all `*TelegramInterface`
```suggestion
interface = PollingTelegramInterface(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
```

Reply:
https://github.com/deeppavlov/dialog_flow_framework/pull/13/commits/d7f45ca9bcf3e6e9d9217680cad5c1bb87e33508